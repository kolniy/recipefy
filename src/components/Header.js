import React from "react";
import { Container } from "reactstrap";

import "../styles/components/header.css";

export const Header = () => {
  return (
    <>
      <section className="section section-shaped section-lg app-header">
        <div className="shape shape-style-1 bg-gradient-info">
          <Container className="app-header__container">
            <h3
              style={{
                color: "#fff",
                textTransform: "uppercase",
                fontWeight: "700",
                fontSize: "30px",
                marginTop: "3rem",
                background: "none",
              }}
            >
              Recipefy
            </h3>
            <p
              style={{
                color: "#fff",
                textTransform: "uppercase",
                fontWeight: "700",
                fontSize: "12px",
                background: "none",
              }}
            >
              Your One Stop Place for Great Recipe Selection.
            </p>
          </Container>
          <span />
          <span />
          <span />
          <span />
          <span />
          <span />
          <span />
          <span />
        </div>
      </section>
    </>
  );
};

export default Header;
