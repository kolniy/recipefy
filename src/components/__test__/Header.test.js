import React from "react";
import { shallow } from "enzyme";
import Header from "../Header";

test("it should render header correctly", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});

test("it should display h3 app title", () => {
  const wrapper = shallow(<Header />);
  const h3 = wrapper.find("h3");
  const result = h3.text();

  expect(result).toBe("Recipefy");
});

test("it should display app subtitle", () => {
  const wrapper = shallow(<Header />);
  const p = wrapper.find("p");
  const paraText = p.text();
  expect(paraText).toBe("Your One Stop App for Great Recipe Selection.");
});
