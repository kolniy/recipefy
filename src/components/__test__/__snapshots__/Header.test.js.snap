// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render header correctly 1`] = `ShallowWrapper {}`;
